model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -708,7 +710,7 @@ class WorkspaceJsonSupport extends JsonSupport {
 
   }
 
-  implicit val MethodConfigurationFormat = jsonFormat10(MethodConfiguration)
+  implicit val MethodConfigurationFormat = jsonFormat11(MethodConfiguration)
```

Comment:
I'm not sure I understand the difference between `jsonFormat11` vs `jsonFormat10`

Reply:
10 parameters in the constructor vs 11 (I added one)